### DISCUSSION TITLE
How use nodemcu8266 instad wemos mini D1

### DISCUSSION THREAD

**Discussion by turbuntux** [philibertc/micronova_controller#55] [Q&A]
I m tring to flash my nodemcu8266, but i m unable to find the network micronova
i don t know how solve.
please helpme.

**Comment by philibertc** [comment]
Hello, 
Do you use this nodemcu?
![image](https://user-images.githubusercontent.com/57588282/209559497-5da243d8-de09-4eb3-996b-c98f521e46f1.png)

I will check, but it should be perfectly compatible. Can you check if you have messages via the serial port?

Best Regards,
Philibert
